superchain/extra/addresses/addresses.json

```diff
@@ -215,15 +215,16 @@
     "OptimismPortalProxy": "0x76114bd29dFcC7a9892240D317E6c7C2A281Ffc6",
     "SystemConfigProxy": "0xa6b72407e2dc9EBF84b839B69A24C88929cf20F7",
     "ProxyAdmin": "0x18d890A46A3556e7F36f28C79F6157BC7a59f867",
-    "DelayedWETHProxy": "0x494E1970923A836Ce00Bdf83dCe4b13981C95039",
+    "SystemConfigOwner": "0x8c20c40180751d93E939DDDee3517AE0d1EBeAd2",
+    "ProxyAdminOwner": "0x4377BB0F0103992b31eC12b4d796a8687B8dC8E9",
+    "Guardian": "0x8c20c40180751d93E939DDDee3517AE0d1EBeAd2",
+    "AnchorStateRegistryProxy": "0x03b82AE60989863BCEb0BbD442A70568e5AefB85",
+    "DelayedWETHProxy": "0xE99696a028171e31a72828A196C27c2Dd670E1aa",
     "DisputeGameFactoryProxy": "0x2419423C72998eb1c6c15A235de2f112f8E38efF",
-    "Mips": "0xCdD4eabeb5f5d1fD195b15AFBF4eC3CA605Ec8Fe",
-    "PreimageOracle": "0xc5BF95A758CA8F0695081d6Ef4F57E204DF2bc4A",
     "FaultDisputeGame": "0x3CdB0e38bC990c07eADA1376248BB2a405Ae3B9B",
+    "Mips": "0xCdD4eabeb5f5d1fD195b15AFBF4eC3CA605Ec8Fe",
     "PermissionedDisputeGame": "0xc06B6A93c4b8ef23e1FB535BB2dd80239ca433AC",
-    "SystemConfigOwner": "0x8c20c40180751d93E939DDDee3517AE0d1EBeAd2",
-    "ProxyAdminOwner": "0x4377BB0F0103992b31eC12b4d796a8687B8dC8E9",
-    "Guardian": "0x8c20c40180751d93E939DDDee3517AE0d1EBeAd2"
+    "PreimageOracle": "0xc5BF95A758CA8F0695081d6Ef4F57E204DF2bc4A"
```

Comment:
Notion doc has `"PreimageOracle": "0xA6662E943C763b23e82eEf3868E50007C9Af10bf",`. Which one is correct?

Reply:
The notion doc one, thanks for flagging . Fixed here [71e54c2](https://github.com/ethereum-optimism/superchain-registry/pull/277/commits/71e54c265316dcb6e7d081264f4a43df498d1eb3)